rollup.config.mjs

```diff
@@ -0,0 +1,15 @@
+import pkg from './package.json';
+
+export default {
+  input: 'index.mjs',
+  external: Object.keys (pkg.dependencies),
+  output: {
+    format: 'umd',
+    name: 'warpState',
+    file: 'index.js',
+    interop: false,
+    globals: {
+      'sanctuary-type-classes': 'Z'
```

Comment:
This is not how Sanctuary Type Classes exposes itself on the global scope. See:

https://github.com/sanctuary-js/sanctuary-type-classes/blob/8c9e2b6ac0ed9d9f1c884014813e3b8e87960b8c/index.js#L81